operator/src/test/java/oracle/kubernetes/operator/helpers/PodHelperTestBase.java

```diff
@@ -196,7 +198,7 @@ V1Container getCreatedPodSpecContainer() {
 
   @Test
   public void whenNoPod_createIt() {
-    expectReadPod(getPodName()).failingWithStatus(HttpURLConnection.HTTP_NOT_FOUND);
+    onAdminExpectListPersistentVolume();
```

Comment:
It looks as though you are adding onAdminExpectListPersistentValue to each and every test method. If so, avoid duplication by adding it to a setUp method.

Reply:
Thanks, I'll look.  That may have happened because I worked on each test case somewhat independently until I got them to work.